linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -681,22 +692,44 @@ pipeline {
               RunPowershellCommand(".\\Extras\\AnalyseAllResults.ps1")
             }
           }
-        }
-        stage('validation_perf_hyperv') {
-          when {
-            beforeAgent true
-            expression { params.ENABLED_STAGES.contains('validation_perf_hyperv') }
-          }
-          agent {
-            node {
-              label 'meta_slave'
+        }  
```

Comment:
do not change the indentation if not needed

Reply:
OK .I`ll try not to touch indentation